tls/s2n_crypto.h

```diff
@@ -77,7 +77,9 @@ struct s2n_crypto_parameters {
     uint8_t client_implicit_iv[S2N_TLS_MAX_IV_LEN];
     uint8_t server_implicit_iv[S2N_TLS_MAX_IV_LEN];
 
+    struct s2n_hash_state signature_hash;
     struct s2n_hmac_state client_record_mac;
+    struct s2n_hmac_state client_record_mac_copy;
```

Comment:
s2n_verify_cbc uses an hmac copy that needs to be allocated and freed at the s2n_connection level. I renamed this to record_mac_copy_workspace for clarity.